README.md

```diff
@@ -187,6 +187,20 @@ Configuration of this plugin should be avoided in local project SBT build files.
 
 No special tasks are enabled for this plugin.
 
+## `CakePublishPlugin`: Artifact Publishing
+
+Plugin requirements: `SonatypePlugin` and `PgpPlugin`
```

Comment:
is this for publishing the `sbt-cake` itself? I was under the impression that we wouldn't want to have artifact/sonatype publishing in typical proprietary projects. It's probably best not to include it by default to avoid accidental mistakes.